src/main/plugin/iso19139.ca.HNAP/loc/fre/labels.xml

```diff
@@ -2423,6 +2423,7 @@
       <option value="HTTPS">HTTPS</option>
       <option value="FTP">FTP</option>
       <option value="ESRI REST: Map Service">ESRI REST: service de carte</option>
+      <option value="ESRI REST: Map Server">ESRI REST: Map Server</option>
```

Comment:
Can you change "Map Server" to "serveur de carte"

Reply:
done.